docs/how-to/h-enable-monitoring.md

```diff
@@ -0,0 +1,215 @@
+# Enable Monitoring
+
+The Charmed Spark solution comes with the [spark-metrics](https://github.com/banzaicloud/spark-metrics) exporter.
+This exporter is designed to push metrics to the [prometheus pushgateway](https://github.com/prometheus/pushgateway), that is integrated with the [Canonical Observability Stack](https://charmhub.io/topics/canonical-observability-stack). 
+
+
+To enable monitor with Charmed Spark just follow this guide.
+
+
+As a prerequisite, you need to have Juju 3 installed with a MicroK8s controller bootstrapped. This can be done following this [tutorial](https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s).
+
```

Comment:
I would just write here that for setting up the observability we need two things:

1. Setup the Observability bundle with juju 
2. Configure the Spark service account 

Then below, you can probably create two subsection, just to make it a bit more structured.